tevm/transport.go

```diff
@@ -119,6 +115,26 @@ func (c *Chain) Execute(req *seth.RPCRequest, res *seth.RPCResponse) error {
 func (c *Chain) execute(method string, params []json.RawMessage) (interface{}, error) {
 	var b blocknum
 	switch method {
+	case "eth_protocolVersion":
+		if err := marshal(params); err != nil {
+			return nil, err
+		}
+		return seth.Uint63(63), nil
```

Comment:
Where does this come from?

Reply:
MyEtherWallet returns that.

Reply:
`curl -X POST -H 'Content-Type: application/json' --data '{"jsonrpc":"2.0","method":"eth_protocolVersion","params":null,"id":1}' https://api.myetherapi.com/eth`